File: CalcIMC/app/src/main/java/com/example/calcimc/MainActivity.java
package com.example.calcimc;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClick(View view)
    {
        Intent it = new Intent  (this, MainActivity2.class);
        Bundle params = new Bundle();
        params.putString("msg", "Olá");
        it.putExtras(params);
        startActivity(it);
    }
}

File: MultiView/app/src/main/java/com/example/multiview/MainActivity.kt
package com.example.multiview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

@Override
public void onCreate (Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContendView(R.layout.activity_tela1);
    }

public void onClick(View view) {
    Intent it = new Intent(this, Tela2.class);
    startActivity(it);
}

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}